app/src/main/java/com/isc/hermes/controller/IncidentFormController.java

```diff
@@ -57,6 +70,25 @@ private void setButtonsOnClick(){
             incidentForm.setVisibility(View.GONE);
             mapController.deleteMarks();
             Toast.makeText(context, "Incident Saved Correctly.", Toast.LENGTH_SHORT).show();
+
+            @SuppressLint("StaticFieldLeak") AsyncTask<Void, Void, Integer> task = new AsyncTask<Void, Void, Integer>() {
+                @Override
+                protected Integer doInBackground(Void... voids) {
+                    return uploadIncidentDataBase();
+                }
+
+                @Override
+                protected void onPostExecute(Integer responseCode) {
+                    if (responseCode == HttpURLConnection.HTTP_OK) {
+                        Toast.makeText(context, "Incident uploaded successfully.", Toast.LENGTH_SHORT).show();
```

Comment:
A small recommendation is that the message `"Incident uploaded successfully."` and `"Incident could not be uploaded."` be in the `"strings.xml"` files instead of adding it here.

Reply:
This observation has been resolved.